tests/zfs-tests/tests/functional/alloc_class/alloc_class_012_pos.ksh

```diff
@@ -33,8 +33,9 @@ function file_in_special_vdev # <dataset> <inode>
 {
 	typeset dataset="$1"
 	typeset inum="$2"
+	typeset num_normal=$(echo $ZPOOL_DISKS | wc -w)
```

Comment:
BSD's `wc` output has a leading tab so this screws up the awk command below. Here's how I usually fix that:
```suggestion
	typeset num_normal=$(echo $ZPOOL_DISKS | wc -w | xargs)
```

Reply:
Thanks for pointing this out!  Would have been very tricky for me to diagnose.